engine/src/main/java/org/terasology/persistence/typeHandling/coreTypes/EnumTypeHandler.java

```diff
@@ -67,10 +69,7 @@ public T deserialize(PersistedData data, DeserializationContext context) {
 
     @Override
     public PersistedData serializeCollection(Collection<T> value, SerializationContext context) {
-        List<String> values = Lists.newArrayListWithCapacity(value.size());
-        for (T item : value) {
-            values.add(item.toString());
-        }
+        List<String> values = value.stream().map(T::toString).collect(Collectors.toCollection(ArrayList::new));
```

Comment:
The method `Collectors.toList()` should do the trick - is there a reason to explicitly specify `ArrayList`?